common/static/common/templates/discussion/forum-actions.underscore

```diff
@@ -1,16 +1,18 @@
-<ul class="<%= contentType %>-actions-list">
-    <% _.each(primaryActions, function(action) { print(_.template($('#forum-action-' + action).html(), {})) }) %>
-    <li class="actions-item is-visible">
-        <div class="more-wrapper">
-            <a href="javascript:void(0)" class="action-button action-more" role="button" aria-haspopup="true" aria-controls="action-menu-<%= contentId %>">
-                <span class="action-label"><%- gettext("More") %></span>
-                <span class="action-icon"><i class="icon fa fa-ellipsis-h"></i></span>
-            </a>
-            <div class="actions-dropdown" id="action-menu-<%= contentType %>" aria-expanded="false">
-              <ul class="actions-dropdown-list">
-                <% _.each(secondaryActions, function(action) { print(_.template($('#forum-action-' + action).html(), {})) }) %>
-              </ul>
+<% if (!readOnly) { %>
```

Comment:
This code is hiding the entire popup menu for read-only users. I wonder whether there are some actions that should still be shown. For example, maybe the "Report" action should be supported for read-only users. @explorerleslie, what's your opinion?